aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -0,0 +1,215 @@
+<%
+require 'cdo/aws/cloudfront'
+require 'aws-sdk'
+%>
+---
+AWSTemplateFormatVersion: 2010-09-09
+Description: AWS CloudFormation Template for Code.org application
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
```

Comment:
There seems to be a fair amount of duplicated network configuration between this and the adhoc template.  Would it make sense to factor out the duplication, or is at a feature that  these configurations will diverge over time?